src/framework/ui/internal/uiconfiguration.cpp

```diff
@@ -328,13 +354,13 @@ ThemeInfo UiConfiguration::makeStandardTheme(const ThemeCode& codeKey) const
     theme.codeKey = codeKey;
 
     if (codeKey == LIGHT_THEME_CODE) {
-        theme.values = LIGHT_THEME_VALUES;
+        theme.values = m_lightThemeValues;
```

Comment:
I think it’s better to make something like:
```
ThemeInfo UiConfiguration::makeStandardTheme(const ThemeCode& codeKey) const
{
    ThemeInfo theme;
    theme.codeKey = codeKey;
    
    Config config = ConfigReader::read(QString(":/configs/%1.cfg").arg(QString::fromStdString(codeKey)));
    theme.values = {
        { BACKGROUND_PRIMARY_COLOR, config.value("background_primary_color").toQString() },
        ...
    }
    
    return theme;
}
```

no need configs and values members of class